### PR TITLE
Create abnormallyPrematureBiologicalProcessInLocation.yaml

### PR DESCRIPTION
**Pull request by pfey03**
seems to look good

### CODE REVIEW

File: src/patterns/dosdp-dev/abnormallyPrematureBiologicalProcessInLocation,yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+annotationProperties:
+  exact_synonym: oio:hasExactSynonym
+
+vars:
+  biological_process: "'biological process'"
+  location: "'independent continuant'"
+
+name:
+  text: "premature %s"
```
Comment (matentzn) [inline src/patterns/dosdp-dev/abnormallyPrematureBiologicalProcessInLocation,yaml:28]: misisng location

File: src/patterns/dosdp-dev/abnormallyPrematureBiologicalProcessInLocation,yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
+pattern_name: abnormallyPrematureBiologicalProcessInLocation
```
Comment (matentzn) [inline src/patterns/dosdp-dev/abnormallyPrematureBiologicalProcessInLocation,yaml:1]: the filename has a comma, not a period :)

File: src/patterns/dosdp-dev/abnormallyPrematureBiologicalProcessInLocation,yaml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+pattern_iri: http://purl.obolibrary.org/obo/upheno/patterns-dev/abnormallyPrematureBiologicalProcess.yaml
```
Comment (matentzn) [inline src/patterns/dosdp-dev/abnormallyPrematureBiologicalProcessInLocation,yaml:2]: iri has wrong name

**Review by matentzn — CHANGES_REQUESTED** [review]

**Review by pfey03 — COMMENTED** [review]
approve of all changes, using patterns already

**Comment by pfey03** [comment]
@matentzn trying to move this forward is it ok now? Thanks!

File: src/patterns/dosdp-dev/abnormallyPrematureBiologicalProcessInLocation,yaml (hunk 3 above)
Comment (matentzn) [reply src/patterns/dosdp-dev/abnormallyPrematureBiologicalProcessInLocation,yaml]: Still wrong: Missing InLocation!

[the remark this answers is not in the document]
Comment (matentzn) [reply src/patterns/dosdp-dev/abnormallyPrematureBiologicalProcessInLocation,yaml]: Thats would be correct!

File: src/patterns/dosdp-dev/abnormallyPrematureBiologicalProcessInLocation.yaml [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+annotationProperties:
+  exact_synonym: oio:hasExactSynonym
+
+vars:
+  biological_process: "'biological process'"
+  location: "'independent continuant'"
+
+name:
+  text: "premature %s"
```
Comment (matentzn) [inline src/patterns/dosdp-dev/abnormallyPrematureBiologicalProcessInLocation.yaml:28]: Still missing location (the text needs to contain %s twice, once or the biological process, once for the location)

**Review by matentzn — CHANGES_REQUESTED** [review]

**Comment by matentzn** [comment]
@pfey03 This still requires some changes from your end to be merged! Thanks :)

File: src/patterns/dosdp-dev/abnormallyPrematureBiologicalProcessInLocation.yaml (hunk 4 above)
Comment (pfey03) [reply src/patterns/dosdp-dev/abnormallyPrematureBiologicalProcessInLocation.yaml:28]: I clicked everywhere but don't know how to edit the file in a branch. I edited the file but it didn't take it and it says I ned to do in branch but how?

**Comment by matentzn** [comment]
Click here: https://github.com/obophenotype/upheno/pull/743/files

Then:

image

**Review by matentzn — APPROVED** [review]

**Pull request merged**